mobile/src/main/java/ch/epfl/sdp/mobile/application/StoreDocuments.kt

```diff
@@ -46,3 +50,116 @@ fun ProfileDocument?.toProfile(currentUserUid: String?): Profile {
 fun ProfileDocument?.toProfile(currentUser: AuthenticationUser): Profile {
   return toProfile((currentUser as? AuthenticatedUser)?.uid)
 }
+
+/**
+ * A document which represents a game of chess between two users. All the game documents are stored
+ * in the `/games/` collection.
+ *
+ * @param moves
+ * @param whiteId
+ * @param blackId
+ */
+data class ChessDocument(
```

Comment:
I'm not sure about this name. Maybe `GameDocument` better reflects what's in the document.

Reply:
Maybe `matchDocument` ?